hcloud/action.go

```diff
@@ -30,6 +31,23 @@ const (
 	ActionStatusError                = "error"
 )
 
+// ResourceReference references other resources from an action.
+type ResourceReference struct {
```

Comment:
After having a look at our API documentation, I think it would be better to name this `ActionResource` since there is no other place where that exact structure is used again.